templates/emails/deactivated.collective.as.host.hbs

```diff
@@ -0,0 +1,13 @@
+Subject: Host De-Activation
+
+{{> header}}
+
+<p>Hi,</p>
+
+<p>You have successfully deactivated your host. Your profile is no longer able to receive and add funds for collectives or receive Bank and PayPal payments. Your profile no longer has a host dashboard either.</p>
```

Comment:
@piamancini I can't process that sentence "Your profile is no longer able to receive and add funds for collectives or receive Bank and PayPal payments."

Reply:
hmmm how about: Your profile is no longer able to manage a collective's budget (receive funds and pay for expenses.)